source/includes/k8s/deploy-operator.rst

```diff
@@ -111,18 +138,60 @@ the example command above may differ from the output in your terminal:
 
 .. important::
 
-   The MinIO Operator automatically generates TLS certificates for all 
-   MinIO Tenant pods using the specified Certificate Authority (CA).
-   Clients external to the Kubernetes cluster must trust the  
-   Kubernetes cluster CA to connect to the MinIO Operator or MinIO Tenants. 
+   The MinIO Operator automatically generates TLS certificates for all MinIO Tenant pods using the specified Certificate Authority (CA).
+   Clients external to the Kubernetes cluster must trust the Kubernetes cluster CA to connect to the MinIO Operator or MinIO Tenants. 
 
-   Clients which cannot trust the Kubernetes cluster CA can try disabling TLS 
-   validation for connections to the MinIO Operator or a MinIO Tenant. 
+   Clients which cannot trust the Kubernetes cluster CA can try disabling TLS validation for connections to the MinIO Operator or a MinIO Tenant. 
```

Comment:
Does `try` here mean it may or may not work? (Not sure if disabling TLS validation is as consequential as disabling TLS generally. Seems like not a great idea however.)

Reply:
Dropped `try` and swapped `disabling` for `disable`